python_modules/dagster/dagster/_core/definitions/asset_layer.py

```diff
@@ -507,6 +507,12 @@ def partitions_fn(context: "OutputContext") -> AbstractSet[str]:
                     )
                     check_handle_by_output[node_output_handle] = check_spec.handle
 
+            for check_handle in assets_def.asset_check_handles:
```

Comment:
any idea why we need this plus the construction above?

Reply:
hmm did I originally add this? if so, I don't remember why it's necessary and it seems like it isn't